crates/ruff/src/autofix/mod.rs

```diff
@@ -38,10 +38,10 @@ fn apply_fixes<'a>(
 
     for (rule, fix) in diagnostics
         .filter_map(|diagnostic| {
-            if diagnostic.fix.is_empty() {
+            if diagnostic.fix.is_none() {
                 None
             } else {
-                Some((diagnostic.kind.rule(), &diagnostic.fix))
+                Some((diagnostic.kind.rule(), diagnostic.fix.as_ref().unwrap()))
             }
         })
```

Comment:
I'm guessing there's a more idiomatic way to handle this now that `diagnostic.fix` is an `Option`?

Reply:
You can write `diagnostic.fix().map(|fix| (diagnostic.kind.rule(), fix))`

Reply:
Wonderful thanks!